packages/roosterjs-content-model/lib/publicTypes/IContentModelEditor.ts

```diff
@@ -53,6 +54,11 @@ export interface DomToModelOption {
      * @default false
      */
     alwaysNormalizeTable?: boolean;
+
+    /**
+     * Checks whether elements needs to be skipped from being processed
+     */
+    shouldSkipProcessElement?: Partial<ShouldSkipProcessElementMap>;
```

Comment:
no need to add this. Just check delimiter directly.

Reply:
Or, another way, you can add a delimiterProcessor, and choose this processor in https://github.com/microsoft/roosterjs/blob/fbf328176e345f697d8514029ef9d3b9a9e48bfa/packages/roosterjs-content-model/lib/domToModel/processors/elementProcessor.ts#L12, then do nothing in this processor, that could be an elegant way to skip delimiter.